decidim-core/app/views/decidim/pages/_standalone.html.erb

```diff
@@ -1,21 +1,17 @@
-<div class="wrapper">
-  <div class="row column">
-    <div class="page-title-wrapper">
-      <h1 class="heading1 page-title">
-        <%= title %>
-      </h1>
-    </div>
+<%= render layout:"layouts/decidim/shared/layout_center" do %>
 
-    <%= cell "decidim/tos_page", :announcement %>
+  <div class="text-center py-12">
+    <h1 class="h1 decorator inline-block text-left">
+      <%= title %>
+    </h1>
+  </div>
 
-    <div class="columns small-12">
-      <div class="card">
-        <div class="card__content"><%= decidim_sanitize_editor translated_attribute page.content %></div>
-      </div>
-    </div>
+  <%= cell "decidim/tos_page", :announcement %>
```

Comment:
The TOS page has currently illogical heading order. Heading orders should be always logical, i.e. h1, h2, h3, h4, h5, h6 always in this order, no other order is allowed.

![TOS page heading order](https://user-images.githubusercontent.com/864340/183089284-ba4995fa-4d9f-48aa-a5ce-6998351c019c.png)

Also as a general note that the `<h1>` should be always the first heading in the DOM. Right now you are using e.g. `<h3>` in the main navigation dropdown which is incorrect as it comes before the `<h1>` on the page. I understand that it is not related particularly to the **pages** redesign but just to point it out regarding this comment about the same problem.